### PR TITLE
Move %_bindir symlinks out from dist-git-selinux

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: dist-git.spec [hunk 1]
```diff
@@ -258,12 +258,13 @@ fi
 
 %dir              %{_datadir}/dist-git
 %attr (775, -, -) %{_datadir}/dist-git/*
+%{_bindir}/*
```
Comment (FrostyX) [inline dist-git.spec:261]: Unrelated but since you are changing the spec, can you please make the `%{_bindir}/*` more specific?

Comment (praiskup) [reply dist-git.spec]: Updated.

**Review by FrostyX — APPROVED** [review]

**Comment by xsuchy** [comment]
@praiskup can you rebase pls? (not sure if this is still relevant after merging your other PR)

**Comment by praiskup** [comment]
I believe this PR is still relevant, PTAL

**Review by xsuchy — APPROVED** [review]

**Pull request merged**